Sources/Public/Animation/View+ValueChanged.swift

```diff
@@ -0,0 +1,20 @@
+// Created by miguel_jimenez on 7/26/23.
+// Copyright © 2023 Airbnb Inc. All rights reserved.
+
+import Combine
+import SwiftUI
+
+@available(iOS 13.0, tvOS 13.0, macOS 10.15, *)
+extension View {
+  /// A backwards compatible wrapper for iOS 14 `onChange`
+  @ViewBuilder
+  func valueChanged<T: Equatable>(value: T, onChange: @escaping (T) -> Void) -> some View {
```

Comment:
Since this is internal, instead of putting this in `Sources/Public` could you move it to `Sources/Private/Utilities/Helpers`?